konig-schemagen/src/main/java/io/konig/showl/WorkbookToTurtleTransformer.java

```diff
@@ -114,7 +113,7 @@ private void writeShapes(File shapesOutDir) throws RDFHandlerException, IOExcept
 		ShapeFileGetter fileGetter = new ShapeFileGetter(shapesOutDir, nsManager);
 		ShapeManager shapeManager = workbookLoader.getShapeManager();
 		VertexCopier copier = new VertexCopier();
-		copier.excludeProperty(SH.shape, SH.path, SH.targetClass, SH.valueClass, Konig.aggregationOf, Konig.rollUpBy, Konig.defaultShapeFor);
+		copier.excludeProperty(SH.shape, SH.path, SH.targetClass, SH.valueClass);

```

Comment:
It looks like your commit is overwriting the fix for Issue #655.
Please revert this change.